Assets/MixedRealityToolkit/Utilities/Editor/AssemblyDefinition.cs

```diff
@@ -0,0 +1,189 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.IO;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Editor
+{
+    /// <summary>
+    /// A class that represents a Unity assembly definition (asmdef) file.
+    /// </summary>
+    [Serializable]
+    public class AssemblyDefinition
+    {
+        /// <summary>
+        /// Creates a new, empty assembly definition.
+        /// </summary>
+        public AssemblyDefinition()
+        { }
+
+        [SerializeField]
+        private string name = null;
+
+        /// <summary>
+        /// Please see <see href="https://docs.unity3d.com/Manual/class-AssemblyDefinitionImporter.html">Assembly Definition properties</see> on the Unity documentation site.
+        /// </summary>
+        public string Name
+        {
+            get => name;
+            set { name = value; }
+        }
+
+        // General
+        [SerializeField]
+        private bool allowUnsafeCode = false;
+
+        /// <summary>
+        /// Please see <see href="https://docs.unity3d.com/Manual/class-AssemblyDefinitionImporter.html">Assembly Definition properties</see> on the Unity documentation site.
+        /// </summary>
+        public bool AllowUnsafeCode
+        {
+            get => allowUnsafeCode;
+            set { allowUnsafeCode = value; }
```

Comment:
These feel like places (throughout this file) where we should match style between `get` and `set` (I'm good with either style!):

![image](https://user-images.githubusercontent.com/3580640/67817924-a7321500-fa6c-11e9-834a-3bd10655d8e9.png)

Reply:
done :)